Source/SwiftLintBuiltInRules/Rules/Lint/PrivateSwiftUIStatePropertyRuleExamples.swift

```diff
@@ -0,0 +1,42 @@
+internal struct PrivateSwiftUIStatePropertyRuleExamples {
+    static let nonTriggeringExamples: [Example] = [
+        Example(
+            """
+            struct ContentView: View {
+                @State private var isPlaying: Bool = false
+            }
+            """
+        ),
+        Example(
+            """
+            struct ContentView: View {
+                @State fileprivate var isPlaying: Bool = false
+            }
+            """
+        ),
+        Example(
+            """
+            struct ContentView: View {
+                var isPlaying = false
+            }
+            """
+        ),
+        Example(
+            """
+            struct ContentView: View {
+                @StateObject var foo = Foo()
+            }
+            """
+        )
+    ]
+
+    static let triggeringExamples: [Example] = [
+        Example(
+            """
+            struct ContentView: View {
+                @State var isPlaying: Bool = false
```

Comment:
The example misses a violation marker `↓` to see where exactly the rule triggers.